src/components/ebay-date-textbox/index.marko

```diff
@@ -1,8 +1,11 @@
+import { format, parseISO } from "date-fns";
+import { getDateFnsLocale, type DayISO } from "../ebay-calendar/date-utils";
 $ const {
     a11yOpenPopoverText = "open calendar",
     range,
-    inputPlaceholderText = "YYYY-MM-DD",
+    inputPlaceholderText = "MM/DD/YYYY",
```

Comment:
We should probably keep the original default, otherwise it will be breaking if they don't pass in `localizeFormat`